### ISSUE TITLE
Validation loss is very low compared to training loss

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/skinstression#2]
The validation loss is very low compared to the training loss (see below)
![image](https://user-images.githubusercontent.com/28396796/193081634-9aea2124-9363-4794-8d69-3158f3baeeab.png)
Why?

Things already checked:
- model.eval()
- torch.no_grad()
- summing the right values

To be tested:
- [x] use training dataset to compare
- [ ] does SubsetRandomSampler really give unique samplers?
- [ ] is it because of bias? We split every image into 4 patches, which we feed the network. Patches are not grouped, so a model trained on the first two patches of an image will perform well on the other two patches of an image. (see #3 )
